src/lib/util.ts

```diff
@@ -47,22 +47,29 @@ const activeConnections: string[] = []
 
 // #region Functions
 
-export function arrayEqual(a: unknown, b: unknown) {
-	if (a === b) return true
+export function checkVariables(firstString: string, secondString: string) {
+	const match = (string: string) =>
+			Array.from(string.matchAll(/%%(\w+)%%|{(\w+)(, \w+(, .+)?)?}/g))
+				.map(el => el[1] ?? el[2])
+				.sort(),
+		arrayEqual = (a: unknown, b: unknown) => {
```

Comment:
Please keep the arrayEqual function separate and sort this function alphabetically within the file